### PR TITLE
Create jameelahmed.md

### PR DESCRIPTION
**Pull request by ahmedj89**

### CODE REVIEW

File: docs/jameelahmed.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Jameel Ahmed
+
+New Delhi, India
+
+Firmware Engineer, Smart Embedded Cotnrols, JCI.
```
Comment (jphelps92) [inline docs/jameelahmed.md]: Change Cotnrols to Controls

**Comment by ahmedj89** [comment]
@jphelps92 Thanks for catching that.

File: docs/jameelahmed.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Jameel Ahmed
```
Comment (cmcleok1) [inline docs/jameelahmed.md]: Add description for each piece of info

Ex: Name: Jameel Ahmed

Comment (ahmedj89) [inline docs/jameelahmed.md]: OK.

**Comment by ahmedj89** [comment]
:+1:

**Comment by HarryPalz** [comment]
I :+1:  as well.

**Pull request merged**